docs/external-template-creation.md

```diff
@@ -84,3 +73,50 @@ export default {
 ```
 
 Once you've completed this step, your new sticker should now appear with the other stickers appended to the bottom of the shapes panel if the `enableStickers` feature flag is enabled.
+
+### Inserting stickers (SVGs) into a story
+
+To add stickers to your template story, go to the `Experiments` page (in the left-hand sidebar under the `Stories` section) and check the `Enable Stickers` checkbox. 
+
+Once turned on, all available stickers in the codebase should be available for selection under our base shapes in the shapes library panel.
+
+### Get the story JSON
+
+To get the JSON representation of a story in the editor:
+
+1. In the editor, open the story.
+2. Open `Chrome DevTools > Network`.
+3. In the editor, click the "Save draft" button. 
+4. You should see a POST XHR with JSON in the request payload. In that payload JSON, find and right-click the `story_data` field and click `Copy value`. 
+5. The story JSON should now be copied to your clipboard and ready to paste into a new file.
+
+**Note**: `Command+Shift+Option+J / Control+Shift+Alt+J` in the editor opens up a dialog you can copy/paste story JSON. However, this is missing some necessary fields for versioning.
```

Comment:
I can't recall what fields are needed that this is missing and the importance of those fields. @choumx or @swissspidy either of you mind running through them and their importance rq for me?

Reply:
is it just the version # field? does our our existing migration script update this fine if there's not one present?

Reply:
The devtools has the version field, so that can't be it. It outputs these fields:

https://github.com/google/web-stories-wp/blob/6308e30c5766e73cdfdd3cf8aca064ee3e4ca264/assets/src/edit-story/components/devTools/devTools.js#L113-L119

That's exactly what the template contains as well:

![Screenshot 2021-04-15 at 20 34 46](https://user-images.githubusercontent.com/841956/114920735-097e0180-9e2a-11eb-8e90-4dd09077c8cc.png)

----

**Note:** `current` is the currently selected page, `selection` the currently selected element(s). Both should actually **not** be set in templates. So these templates should be fixed there. That's another thing we can catch in tests/lints). ⚠️ 

**Note 2**: `globalStoryStyles` actually refers to globally set styles in the user's database. These should **not** be set for templates, and really overridden upon loading. This is quite the bug! If you create a template on site X with 1000 global styles, and someone uses that template, they will see all those 1000 global styles instead of their own. ⚠️ 

----

What I just did:

1. Click on "use template" to load the beauty template in the editor
2. Open our devtools and copy the JSON data.
3. Replace contents of `beauty.json` with JSON data
4. Run Prettier

Besides the `globalStoryStyles` data (which is clearly a bug as per above!) and the image URLs, there were **no** differences.

So the only diffs are like this:


```diff
-            "src": "__WEB_STORIES_TEMPLATE_BASE_URL__/images/templates/beauty/beauty_page1_bg-alt.png",
+            "src": "https://wp.stories.google/static/main/images/templates/beauty/beauty_page1_bg-alt.png",
```

So I don't see why the devtools don't work for this?

Certainly way easier than trying to fiddle with Chrome DevTools!

Reply:
awesome! yea the popup modal is nice. 

yea we should reset all the other fields beyond `pages` and `version` . Not sure which fields (if any) we already are doing that for, but happy to create a ticket for it.

Thanks for running me through this!

Reply:
Created #7227 for the fieldsreset.

Reply:
tl;dr let's advocate for using the devtools popup modal

Reply:
Done. Kept the devtools steps as a backup/alternative for now.